src/Esprima/ParserOptions.cs

```diff
@@ -53,5 +53,10 @@ public ParserOptions(IErrorHandler errorHandler)
         /// Gets or sets whether the Regular Expression syntax should be converted to a .NET compatible one.
         /// </summary>
         public bool AdaptRegexp { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets jsx parser.
+        /// </summary>
+        public bool JSX { get; set; } = false;
```

Comment:
Maybe too simple of a name (maybe only)

Reply:
Is there any suggestion for naming cause I'm not good at.